src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -3271,7 +3271,7 @@
     <value>Files is running as administrator</value>
   </data>
   <data name="FilesRunningAsAdminContent" xml:space="preserve">
-    <value>Due to platform limitations, drag and drop isn't available when running Files as administrator.</value>
+    <value>Due to an issue with Windows, drag and drop isn't available when running Files as admin. If you want to use drag and drop and have opened Files normally, Open UAC (User Account Control) from the start menu then move the slider to 3rd level from the bottom then restart Windows. Otherwise, you can keep using Files without drag & drop.</value>
```

Comment:
```suggestion
    <value>Due to an issue with Windows, drag and drop isn't available when running Files as admin. If you want to use drag and drop and have opened Files normally, Open UAC (User Account Control) from the start menu then move the slider to 3rd level from the bottom then restart Windows. Otherwise, you can keep using Files without drag &amp; drop.</value>
```